### PR TITLE
7/dockerfile

### PR DESCRIPTION
**Pull request by StefanCaldararu**
This container now works for me for gymchrono. Please make sure it works for you guys and lmk how the docs look.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- [Attaching to the Container](#attaching-to-the-container)
+- [Running a GymChrono Test](#running-a-gymchrono-test)
+
+## Prerequisites
+
+To be able to follow this quickstart guide, you must have:
+- A modern browser installed (such as chrome)
+- A copy of this repository cloned on a local machine
+- [Docker](https://www.docker.com/get-started/) installed on your system
+- The [OptiX 7.7.0](https://www.docker.com/get-started/) script for linux downloaded and placed in the `/gym-chrono/docker/optix/` directory
```
Comment (Huzaifg) [inline docker/README.md]: Optix link points to docker

**Comment by Huzaifg** [comment]
Is this the error when quotas are reached?
image

**Comment by Huzaifg** [comment]
I tested the instructions in the readme. I ran into 2 problems
1) As posted above, the gdrive quota ran out and I had to run the `gymChrono.sh` script within the container. For now I think this is okay, but long term when some of us gets the time, we should think of another solution to store the data on something that does not have quota limits.

2) I could not run the container after building it with this error
`95a0408ae8006283f137fd849a6cddecddebdf5ba2a2343f623757c154a41145
docker: Error response from daemon: could not select device driver "" with capabilities: [[gpu]].
`
I then had to install the NVIDIA Docker toolkit and configure it
```bash
sudo apt-get install -y nvidia-container-toolkit
sudo nvidia-ctk runtime configure --runtime=docker
sudo systemctl restart docker
```
I am not sure what we can do about this other than adding another note about this

**Comment by StefanCaldararu** [comment]
> I tested the instructions in the readme. I ran into 2 problems
> 
> 1. As posted above, the gdrive quota ran out and I had to run the `gymChrono.sh` script within the container. For now I think this is okay, but long term when some of us gets the time, we should think of another solution to store the data on something that does not have quota limits.
> 2. I could not run the container after building it with this error
>    `95a0408ae8006283f137fd849a6cddecddebdf5ba2a2343f623757c154a41145 docker: Error response from daemon: could not select device driver "" with capabilities: [[gpu]]. `
>    I then had to install the NVIDIA Docker toolkit and configure it
> 
> ```shell
> sudo apt-get install -y nvidia-container-toolkit
> sudo nvidia-ctk runtime configure --runtime=docker
> sudo systemctl restart docker
> ```
> 
> I am not sure what we can do about this other than adding another note about this

Fixed the README for `NVIDIA CONTAINER TOOLKIT`, will open an issue about data download so we can fix it later.

**Pull request merged**